syntax/verilog_systemverilog.vim

```diff
@@ -288,7 +240,8 @@ if index(s:verilog_syntax_fold, "block_nested") >= 0 || index(s:verilog_syntax_f
             \ contains=TOP
         syn match verilogStatement "\<begin\|end\>"
     endif
-elseif index(s:verilog_syntax_fold, "block") >= 0 || index(s:verilog_syntax_fold, "all") >= 0
+elseif index(s:verilog_syntax_fold, "block") >= 0 ||
+    \ index(s:verilog_syntax_fold, "all") >= 0
```

Comment:
If your pane only has 80 columns then I'd argue that this is more readable as you can see that there are no extra terms hiding. No worries though, I don't mind changing it back. I added `let g:vim_indent_cont = &sw` to change the default indentation.